doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -338,7 +338,9 @@ Wi-Fi samples
 Other samples
 -------------
```

Comment:
Consider aligning commit message (you no longer use the default NCS sample configuration)